README.md

```diff
@@ -1,4 +1,4 @@
-# <img src="https://raw.githubusercontent.com/bareos/bareos/master/webui/public/img/bareos.png" alt="Bareos" />
+# <img src="https://raw.githubusercontent.com/bareos/bareos/master/webui/public/themes/default/img/bareos.png" alt="Bareos" />
 
 [![Build Status](https://travis-ci.com/bareos/bareos.svg?branch=master)](https://travis-ci.com/bareos/bareos/branches)
```

Comment:
```suggestion
```
Travis is no longer used.

Reply:
Removed, see https://github.com/bareos/bareos/pull/1281/commits/8970e82ef1270b25872431d73ee08cab33dece63 .

Reply:
This conversation can be closed, I think.